src/mvAppItem.cpp

```diff
@@ -4234,6 +4355,10 @@ DearPyGui::GetEntityParser(mvAppItemType type)
         args.push_back({ mvPyDataType::UUID, "colormap", mvArgType::KEYWORD_ARG, "0", "mvPlotColormap_* constants or mvColorMap uuid from a color map registry" });
         args.push_back({ mvPyDataType::Float, "min_scale", mvArgType::KEYWORD_ARG, "0.0", "Sets the min number of the color scale. Typically is the same as the min scale from the heat series." });
         args.push_back({ mvPyDataType::Float, "max_scale", mvArgType::KEYWORD_ARG, "1.0", "Sets the max number of the color scale. Typically is the same as the max scale from the heat series." });
+        args.push_back({mvPyDataType:: String, "format", mvArgType::KEYWORD_ARG, "'%g'", "Formatting used for the labels."});
+        args.push_back({mvPyDataType:: Bool, "invert", mvArgType::KEYWORD_ARG, "False", "invert the colormap bar and axis scale (this only affects rendering; if you only want to reverse the scale mapping, make scale_min > scale_max)"});
+        args.push_back({mvPyDataType:: Bool, "no_label", mvArgType::KEYWORD_ARG, "False", "the colormap axis label will not be displayed"});
+        args.push_back({mvPyDataType:: Bool, "opposite", mvArgType::KEYWORD_ARG, "False", "render the colormap label and tick labels on the opposite side"});
```

Comment:
`opposite` - This one also feels a little bit confusing. Here's my concern: when one encounters such an argument on a plot axis, it's clear that the axis is to be drawn on the opposite side of the plot, i.e. there's a thing for the widget to be opposite to. However, in case of the color map scale, `opposite` suggests that the scale itself should be opposite - to what?

I'm not sure what would be a better name - `axis_opposite`?